bindata/network/ovn-kubernetes/managed/alert-rules-control-plane.yaml

```diff
@@ -117,3 +117,25 @@ spec:
       for: 15m
       labels:
         severity: warning
+    - alert: OVNKubernetesNorthboundDatabaseTermLag
+      annotations:
+        summary: OVN northbound databases RAFT term have not been equal for a period of time.
+        description: OVN northbound database(s) RAFT term have not been equal which may indicate degraded OVN database high availability.
+      expr: |
+        # Without max_over_time, failed scrapes could create false negatives, see
+        # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
+        max(max_over_time(ovn_db_cluster_term{db_name="OVN_Northbound"}[5m])) - min(max_over_time(ovn_db_cluster_term{db_name="OVN_Northbound"}[5m])) > 0
+      for: 25m
+      labels:
+        severity: warning
+    - alert: OVNKubernetesSouthboundDatabaseTermLag
+      annotations:
+        summary: OVN southbound databases RAFT term have not been equal for a period of time.
```

Comment:
Its a RAFT term. A term indicates a period of time. It increments. A new term value is created when there is a new candidate attempting to be a leader of the RAFT cluster. The winning candidate sets the term value and the followers, if they accept it, also adopt this term value. Each of the RAFT cluster members store this term value and it helps them detect stale information.
This alert, will alert if there are any cluster members who are persistently not agreeing on the term number. It can help detect split brain.

Reply:
How frequently these conditions are checked?  I see that interval here is 25 minutes (+5 ?), during that time databases may transfer leadership up to 9 times, that will trigger re-elections and term changes.  Can this cause false-positive alerts?

Reply:
Either every 30s or 1m. I cannot remember which.
I checked it on a large cluster > 100 nodes after multiple scale runs and it was ok.
I artificially induce leadership transfer 20 times over 30 minutes and check the results.

Reply:
should the summary be "OVN Southbound DB Raft Term not incrementing over time" ?

Reply:
I think my summary is better. We want to detect if term values haven't been the same for a period of time and not that they are incrementing over time.

Reply:
I performed this test by repeatedly deleting the SB DBs RAFT leader container. 
I saw the leader changing and the term becoming consistent between all the new followers within a max time of 30 seconds, usually it was quicker.
The alert promql expression didn't become true during the entire event.
If it did become true, It would need to hold true at least another 30 times in a row.